Source/Scene/MapboxImageryProvider.js

```diff
@@ -21,14 +21,7 @@ define([
     'use strict';
 
     var trailingSlashRegex = /\/$/;
-    var defaultCredit1 = new Credit({
-        text:'© Mapbox © OpenStreetMap',
-        link: 'https://www.mapbox.com/about/maps/'
-    });
-    var defaultCredit2 = [new Credit({
-        text: 'Improve this map',
-        link: 'https://www.mapbox.com/map-feedback/'
-    })];
+    var defaultCredit = new Credit('<a href="https://www.mapbox.com/about/maps/" target="_blank">© Mapbox © OpenStreetMap</a> <a href="https://www.mapbox.com/map-feedback/">Improve this map</a>');
```

Comment:
Shouldn't we just use `&copy;` for the copyright symbol now?

Reply:
This is also incorrect, see https://www.mapbox.com/help/how-attribution-works/#text-attribution